src/gcontrols/web/tabs_compile.html

```diff
@@ -2,49 +2,51 @@
     ATTR-IF{{style="attributes.style.value"}}
     ng-class="['attributes.dynamicClasses.value', 'attributes.classes.value', 'attributes.whiteframe.value']" flex="100" layout="column">
     <div ng-show="attributes.toolbar.visible.value"><div dfx-gcc-web-toolbar></div></div>
-    <md-content flex="100" ATTR-IF{{style="attributes.bodyStyle.value"}} ATTR-IF{{ng-hide="attributes.toolbar.collapsed.value"}} layout="column">
-        <md-tabs md-no-ink-bar="ATTR-EXP{{attributes.inkBar.value==='disabled' ? 'true': 'false'}}ATTR-ENDEXP"
-                 ATTR-IF{{md-center-tabs="attributes.centerTabs.value"}}
-                 md-dynamic-height md-border-bottom md-selected="attributes.tabIndex.value"
-                 ATTR-IF{{md-align-tabs="attributes.tabAlign.value"}}
-                 ATTR-IF{{md-stretch-tabs="attributes.stretching.value"}}
-                 ATTR-IF{{ng-click="attributes.onclick.value"}}
-                 ATTR-IF{{ng-dblclick="attributes.ondblclick.value"}}
-                 ATTR-IF{{ng-keypress="attributes.onkeypress.value"}}
-                 ATTR-IF{{ng-keydown="attributes.onkeydown.value"}}
-                 ATTR-IF{{ng-keyup="attributes.onkeyup.value"}}
-                 ATTR-IF{{ng-mouseover="attributes.onmouseover.value"}}
-                 ATTR-IF{{ng-mouseenter="attributes.onmouseenter.value"}}
-                 ATTR-IF{{ng-mousemove="attributes.onmousemove.value"}}
-                 ATTR-IF{{ng-mouseleave="attributes.onmouseleave.value"}}
-                 ATTR-IF{{ng-mousedown="attributes.onmousedown.value"}}
-                 ATTR-IF{{ng-mouseup="attributes.onmouseup.value"}}
-                 flex="100" layout="column">
+    <md-content id="{{component_id}}_content" flex="100" ATTR-IF{{style="attributes.bodyStyle.value"}} ATTR-IF{{ng-hide="attributes.toolbar.collapsed.value"}} layout="column">
+        <div>        
```

Comment:
This new <div> should have flex="100" and layout="column", like md-content above, otherwise, it will crash flex chain in DOM structure.